src/zenml/utils/materializer_utils.py

```diff
@@ -185,3 +186,15 @@ def _load_artifact(
     logger.debug("Artifact loaded successfully.")
 
     return artifact
+
+
+def get_python_version() -> str:
```

Comment:
Can we reuse `Environment.python_version()` for this instead? Should be the same without the `python` prefix